systers_portal/meetup/views.py

```diff
@@ -19,3 +20,18 @@ class MeetupLocationList(ListView):
     template_name = "meetup/list_location.html"
     model = MeetupLocation
     paginate_by = 20
+
+
+class MeetupView(MeetupLocationMixin, DetailView):
```

Comment:
I did tried some of such options like setting `model` changing `getquery` to `get_queryset` and `change_context_data` but they gave same error or 404 error.

Also in first approach, for wrong pair(of meetup_location and meetup while testing) I am getting `Exception Type: 	DoesNotExist` and `Exception Value: 	Meetup matching query does not exist.` Please can you suggest me how to test this? I am trying to use `self.assertRaises(Meetup.DoesNotExist, self.client.get(incorrect_pair_url))` inside test but still it is failing and giving error `meetup.models.DoesNotExist: Meetup matching query does not exist.` as result of test.